src/routes/create/+page.svelte

```diff
@@ -77,6 +80,23 @@
   function handleAccountNameInput(event: Event) {
     const target = event.target as HTMLInputElement;
     const name = target.value.trim().toLowerCase();
+    
+    // Validate account name length
+    if (name.length > 9) {
+      error.set($t("Account name must be 9 characters or less"));
+      nameAvailable.set(false);
+      accountName.set(name);
+      return;
+    }
+
+    // Check if name contains only allowed characters (a-z and 1-5)
+    if (!/^[a-z1-5]*$/.test(name)) {
```

Comment:
Probably better to use something like this to determine if it follows all the rules, rather than just replicating part of them.

```
export function validName(value: string) {
	try {
		const name = Name.from(value);
		if (value && String(name) === value) {
			return true;
		}
		return false;
	} catch (e) {
		return false;
	}
}
```

Reply:
I thought of doing something like this, but I figured it was better to return more specific error messages rather than one generic one.

Reply:
I replaced the second check with your function though. That doesn't hurt the UX 😄